mindsdb/interfaces/model/model_controller.py

```diff
@@ -29,6 +30,15 @@
 IS_PY36 = sys.version_info[1] <= 6
 
 
+def delete_model_storage(model_id, ctx_dump):
+    try:
+        ctx.load(ctx_dump)
+        modelStorage = ModelStorage(model_id)
+        modelStorage.delete()
```

Comment:
if we are having a huge amount of versions, maybe worth implementing deleting json values ModelStorage.delete (using 'resource_id in ...' filter) ?